apps/style/common.scss

```diff
@@ -1354,7 +1354,7 @@ $workspace-header-button-margin: ($workspace-headers-height -
 .invisible {
   visibility: hidden;
 }
-.hide-source {
+.hide-instructions {
```

Comment:
I wish this was all done in React :)

I can peek through git history and see if there are any further clues about this.